playground/src/PlaygroundView.tsx

```diff
@@ -183,6 +188,39 @@ export class PlaygroundView extends React.Component<IPlaygroundViewProps, IPlayg
     );
   }
 
+  private _renderSelectSample(): React.ReactNode {
+    return (
+      <select
```

Comment:
Consider adding a `<label htmlFor>Samples:</label>` or `aria-label` on `<select>` depending on if you want it visible to avoid an accessibility error.

![image](https://user-images.githubusercontent.com/706967/46762290-8f35fa80-cc8b-11e8-916c-8c00e2b4667d.png)